samples/teamsfx-app/package.json

```
@@ -0,0 +1,59 @@
+{
+  "name": "teamsfx-app",
+  "version": "0.1.0",
+  "engines": {
+    "node": "16 || 18"
+  },
+  "private": true,
+  "dependencies": {
+    "@fluentui/react-components": "^9.18.0",
+    "@microsoft/mgt-element": "*",
+    "@microsoft/mgt-react": "*",
+    "@microsoft/mgt-teamsfx-provider": "*",
+    "@microsoft/microsoft-graph-client": "^3.0.1",
+    "@microsoft/teams-js": "^2.7.1",
+    "@microsoft/teamsfx": "^2.2.0",
+    "@microsoft/teamsfx-react": "^3.0.0",
+    "axios": "^0.21.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.8.0",
+    "react-scripts": "^5.0.1"
+  },
+  "devDependencies": {
+    "@types/node": "^14.0.0",
+    "env-cmd": "^10.1.0",
+    "typescript": "^4.1.2"
+  },
+  "peerDependencies": {
+    "@types/react": "^18.0.0",
+    "@types/react-dom": "^18.0.0",
+    "@types/react-router-dom": "^5.3.3"
+  },
```

The sample wouldn't spin up using F5 until I moved these peer dependencies to dev dependencies.
I did just get a notification that to checkout what changed with teams toolkit 5.1, so that might be the source.

```suggestion
  "devDependencies": {
    "@types/node": "^14.0.0",
    "@types/react": "^18.0.0",
    "@types/react-dom": "^18.0.0",
    "@types/react-router-dom": "^5.3.3",
    "env-cmd": "^10.1.0",
    "typescript": "^4.1.2"
  },
```